crowdfunding_compassion/controllers/homepage_controller.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime
+
+from odoo.http import request, route, Controller
+
+
+class HomepageController(Controller):
+    @route("/home", auth="public", website=True)
```

Comment:
The module should have all routes starting with /together as a prefix for the platform.
```suggestion
    @route("/together/home", auth="public", website=True)
```